tests/parametric/test_config_consistency.py

```diff
@@ -0,0 +1,52 @@
+"""
+Test configuration consistency for functions among different languages for APM.
+"""
+import pytest
+
+from utils import scenarios
+
+parametrize = pytest.mark.parametrize
+
+TEST_SERVICE = "test_service"
+TEST_ENV = "test_env"
+DEFAULT_ENVVARS = {
+    "DD_SERVICE": TEST_SERVICE,
+    "DD_ENV": TEST_ENV,
+}
+
+@scenarios.parametric
+class TestTraceEnabled:
+    @parametrize(
+        "library_env", [{**DEFAULT_ENVVARS, "DD_TRACE_ENABLED": "true"},],
+    )
+    def test_tracing_enabled(self, library_env, test_agent, test_library):
+        trace_enabled_env = library_env.get("DD_TRACE_ENABLED") == "true"
+        if trace_enabled_env:
```

Comment:
```suggestion
        trace_enabled = library_env.get("DD_TRACE_ENABLED") == "true"
        if trace_enabled:
```
This name seems to more accurately reflect what this variable represents.